sky/resources.py

```diff
@@ -204,7 +204,12 @@ def _set_accelerators(
                     except ValueError:
                         with ux_utils.print_exception_no_traceback():
                             raise ValueError(parse_error) from None
-            assert len(accelerators) == 1, accelerators
+
+            # Ignore check for the local cloud case.
+            # It is possible the accelerators dict can contain multiple
+            # types of accelerators for some on-prem clusters.
+            if not isinstance(self._cloud, clouds.Local):
+                assert len(accelerators) == 1, accelerators
```

Comment:
just to confirm: do you mean the local cluster can have 2+ types of GPUs?

Reply:
Yes, one node can have many types of GPUs (i.e. in Ambi's case)